docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+# Asynchronous Workflows
+
+Asynchronous workflows describe situations where the loop between measurements and
+recommendations is more complex and needs to incorporate various other aspects. These
+could for instance be:
+- **Distributed workflows**: When recommendations are distributed across several
+  operators, e.g. at different locations or in several reactors, some experiments might
+  have been started, but are not ready when the next recommendation is requested.
+  Without further consideration, the algorithm would be likely to recommend the pending
+  measurements again, as it is unaware they were already started.
```

Comment:
perhaps add rationale why this is the case -> because, after all, they have been considered to be most promising

Reply:
imo not very needed + find it hard to include that int his sentence in addition to the other stuff please make suggestion

Reply:
Agree with Martin here, is imo not needed.

Reply:
adrians response pending 4w, 1 more day otherwise I will yank this comment

Reply:
OK, if you both think it is obvious to the reader (hard for me to tell, but I always assume the worst 😄 ) then let's leave it. My only way to include it would have been in the direction of `Without further consideration, the algorithm would be likely to recommend the pending measurements again (since they were and still are considered most promising), as it is unaware they were already started.`

By the way, terminology: I notice that we are still talking about `pending measurement` in the userguide, not `pending experiment` like we do it in the code. Shall we harmonize?

Reply:
included `(since they were and still are considered most promising)` which hopefully should satisfy everyone